src/components/InvoicesList/InvoicesList.js

```diff
@@ -125,8 +125,14 @@ const InvoicesList = ({ dependencies: { dopplerBillingApiClient } }) => {
                     {stateInvoices.items.map((invoice, index) => {
                       return (
                         <tr key={index}>
-                          <td>{invoice.accountId}</td>
-                          <td>{invoice.product}</td>
+                          <td>
+                            {invoice.documentType === 'FC' ? (
+                              <FormattedMessage id="invoices_list.invoice_text" />
+                            ) : (
+                              <FormattedMessage id="invoices_list.credit_note_text" />
+                            )}
```

Comment:
What do you think about making literal keys dependant on the document type?
```suggestion
                            <FormattedMessage id="invoices_list.document_{invoice.documentType}" />
```

Reply:
Yes, for me is ok. Instead of having ```"invoices_list.invoice_text"``` and ```"invoices_list.credit_note_text"``` we would have ```"invoices_list.document_FC"``` and ```"invoices_list.document_NC"```

Reply:
Done